translations/messages+intl-icu.fr.yml

```diff
@@ -1480,3 +1480,9 @@ administrator:
             communautes_thematiques: Communautés thématiques
             elections_departementales: Élections départementales
             archives: Archives
+
+chatbot:
+    type:
+        telegram: Telegram
+    assistant_type:
+        openai: OpenAI
```

Comment:
what about making it in two lines ?

```
chatbot.type.telegram:
chatbot.assistant_type.openai:
```

it's easier to search when it's in one line